app/controllers/pages.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+get '/' do
+  redirect '/dashboard' if session[:email]
+  erb :index
+end
+
+get '/dashboard' do
+  require_admin
+  @user = User.first(email: session[:email])
+  @digests = @user.userdigests.reverse
+  @digest = @digests[0]
```

Comment:
we have a ruby method called `first` so `@digests[0]` could be `@digests.first` which IMO sounds more readable